### ISSUE TITLE
Install fails on Yosemite

### ISSUE THREAD

**Issue by abhi18av** [jupyter/atom-notebook#52]
Hi!

I'm a huge fan of the work you are doing and it's really amazing.

The issue is that I can't install it in my system. I've definitely explored the node-gyp / python2 suggestion but this still doesn't work.

Any suggestions, how I can solve it.

`bash
apm  1.9.2
npm  2.13.3
node 0.10.40
python 3.5.1
git 2.8.1

`

`node-gyp --python /usr/bin/python`

**Comment by sadovnychyi** [comment]
I guess you have to have xcode installed?

```
xcode-select: error: tool 'xcodebuild' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance
```

**Comment by abhi18av** [comment]
Nope, that wasn't the issue I already did install the entire toolkit. I guess the path was setup in the wrong way. node-gyp relies too much on python2.

Thanks for looking it up.

**Comment by gnestor** [comment]
This repo is not being actively maintained 🙁 In the meantime, I suggest that you look at [hydrogen](https://github.com/nteract/hydrogen) to run code inline in Atom's text editor, [nteract](https://github.com/nteract/nteract) (a new Jupyter notebook client built using Electron, React, Redux, RxJS), [Jupyter Notebook](https://github.com/jupyter/notebook), and [JupyterLab](https://github.com/jupyterlab/jupyterlab). 

Having said that, nteract is currently extracting its many components into separate packages which will allow them to be reused and assembled to create new apps. When I have the time, I would like to use them to completely refactor this project.
